interfaces/matlab_experimental/readme.md

```diff
@@ -5,30 +5,25 @@ function calling from Cantera CLib.
 
 ## Installation guide:
 
-1. Install Matlab (any release newer than R2008a).
+1. Install Matlab (any release newer than R2008a), then choose one of the following
+   methods to obtain the Cantera CLib and header files for the toolbox:
 2. Compile Cantera from Source and install in your Conda environment, as directed in
-   this link. https://cantera.org/stable/develop/index.html. The
-   experimental Matlab Toolbox does not require a SCons option to install at this moment
-   since it's stand-alone.
-3. For first time users, launch Matlab, then navigate to `/path/to/cantera/source/code`
-   (the folder containing `interfaces` and `samples`) using "Browse for Folder".
-4. For Linux users: Matlab currently uses Intel MKL which uses 64-bit integer types
+   this link: https://cantera.org/stable/develop/index.html. OR
+3. Install the Cantera development interface, as instructed in this link:
+   https://cantera.org/stable/install/conda.html#development-c-fortran-90-interface.
+4. For first time users, launch Matlab, then navigate to
+   `/path/to/cantera/matlab/toolbox` using "Browse for Folder".
+5. For Linux users: Matlab currently uses Intel MKL which uses 64-bit integer types
    that are incompatible with the standard 32-bit integers used by the default version
    of OpenBLAS that comes with Cantera. As such, the correct environment variables
    need to be set to launch Matlab with the correct BLAS/LAPACK libraries loaded:
    `export LD_PRELOAD=/path/to/openblas/library:/path/to/lapack/library`
    then, launch Matlab in the terminal with:
    `matlab -softwareopengl`.
-5. In the Matlab command window, run
-   `addpath(genpath([pwd, '/interfaces/matlab_experimental']))` to add search path for
-   the experimental toolbox.
-6. In the Matlab command window, run
-   `cd([pwd, '/interfaces/matlab_experimental/Utility'])` to navigate to the Utility
-   folder.
-7. Open the file named 'ctRoot.m', in the second line, edit `output=` to
-   `output='/path/to/conda/environment'`, then save the file. This sets the search path
-   for the `ctLoad` command to find the shared library file for Cantera.
-8. In the Matlab command window, run `savepath` to save all search paths.
+6. Run the built-in Utility function `ctPaths('Set', envPath)` to set search paths
+   for the Toolbox:
+   - `envPath`: this is the location of the Conda environment where the CLib and header
```

Comment:
What if the user installed via pip? If current instructions are limited to conda environments, then the instructions should state this.

Reply:
Pip only installs the Python package, not a full version of the Cantera library with header files. Beyond that, though, I think some of what is done here currently is specific to the Conda directory layout, and won't work for a locally-copied version of Cantera (at least not on all platforms).